docs/commands/mcfunctions.md

```diff
@@ -40,26 +39,23 @@ Functions are useful in many ways to reduce the time spent going from command bl
 
 ```yaml
 #Spawn Effects
-effect @a [tag=atSpawn] regeneration 12 255 true
-effect @a [tag=atSpawn] saturation 12 255 true
-effect @a [tag=atSpawn] weakness 12 255 true
+/effect @a [tag=atSpawn] regeneration 12 255 true
```

Comment:
I'd prefer without slashes.

Reply:
I agree.
It was mostly added for consistency with other pages. Also to convey that using ` / ` in functions is not an issue (mentioned just below that.)

The reason other pages include ` / ` is so that beginners can identify different commands in the system easier.

Are you sure I should update it? 👀